README.md

```diff
@@ -8,15 +8,15 @@ is so you always ends up with the same binary, every time.
 
 # Synopsis
 ```
-./kong-ngx-build -p buildroot --openresty 1.13.6.2 --openssl 1.1.1b --luarocks 3.0.4 --debug
+./kong-ngx-build -p buildroot --openresty 1.13.6.2 --openssl 1.1.1b --luarocks 3.0.4 --pcre 8.41 --debug
```

Comment:
Suggested ver should be 8.43 (latest)

Reply:
interesting I don't think that's what we're using currently 🤔

Reply:
Nonetheless?

Reply:
yup I agree 💯 and done